python/tvm/meta_schedule/testing/tune_relay_meta_schedule.py

```diff
@@ -133,19 +136,21 @@ def main():
         params=params,
     )
     graph, rt_mod, params = lib.graph_json, lib.lib, lib.params
-    if input_dtype.startswith("float"):
-        input_data = np.random.uniform(size=input_shape).astype(input_dtype)
-    else:
-        input_data = np.random.randint(low=0, high=10000, size=input_shape, dtype=input_dtype)
+    for input_name, input_shape in input_info.items():
+        if input_dtype.startswith("float"):
+            inputs.append(np.random.uniform(size=input_shape).astype(input_dtype))
+        else:
+            inputs.append(np.random.randint(low=0, high=10000, size=input_shape, dtype=input_dtype))
 
-    def f_timer(rt_mod, dev, input_data):
```

Comment:
`input_data` sounds more reasonable to me than just `inputs` but I don't have very strong opinion here.